src/pages/home/HomePage.vue

```diff
@@ -171,8 +180,16 @@ export default defineComponent({
 
         &--activated {
             background-color: white;
-            color: $link-blue;
+            color: var(--link-color);
         }
     }
+
+    // guarantees wallet connet on top of footer
+    &__footer {
+        z-index: 2;
+    }
+    &__connect-wallet {
+        z-index: 3;
+    }
```

Comment:
`z-index`s should be added to a new file called `_z-index.scss` so we can visualize all of them in one place. i am guilty of not doing this in `AppNav.scss` because we didnt have our styles set up correctly ... can you make this change?

Reply:
Yes, with pleasure! <3